src/app/tests/TestIcdMonitoringTable.cpp

```diff
@@ -93,17 +99,23 @@ void TestSaveAndLoadRegistrationValue(nlTestSuite * aSuite, void * aContext)
     NL_TEST_ASSERT(aSuite, kTestFabricIndex1 == entry.fabricIndex);
     NL_TEST_ASSERT(aSuite, kClientNodeId11 == entry.checkInNodeID);
     NL_TEST_ASSERT(aSuite, kClientNodeId12 == entry.monitoredSubject);
-    NL_TEST_ASSERT(aSuite, sizeof(kKeyBuffer1a) == entry.key.size());
-    NL_TEST_ASSERT(aSuite, 0 == memcmp(entry.key.data(), kKeyBuffer1a, entry.key.size()));
+
+// Cannot compare nor retrieve Data if using PSA crypto
+#if !CHIP_CRYPTO_PSA
```

Comment:
Do the tests run with PSA crypto?

Reply:
Yes for the ARM CI, but you cannot compare a key with PSA_CRYPTO